src/main.cpp

```diff
@@ -305,21 +305,44 @@ static std::optional<struct timespec> get_boottime()
     bpftrace.cat_bytes_max_ = proposed;
   }
 
-  if (!get_bool_env_var("BPFTRACE_NO_USER_SYMBOLS",
-                        bpftrace.resolve_user_symbols_,
-                        true))
-    return false;
-
-  if (!get_bool_env_var("BPFTRACE_CACHE_USER_SYMBOLS",
-                        bpftrace.cache_user_symbols_))
-    return false;
+  if (const char* env_p = std::getenv("BPFTRACE_CACHE_USER_SYMBOLS"))
+  {
+    std::string s(env_p);
+    // Note: options 0 and 1 are for compatibility with older versions of
+    // BPFtrace
```

Comment:
```suggestion
    // bpftrace
```
The old name really refuses to go away! 😂